docs/sdks/server-sdks/go/bandits.mdx

```diff
@@ -0,0 +1,259 @@
+</ApiOptionRef>
+
+<ApiOptionRef 
+  name="SubjectNumericAttributes"
+  type="map[string]float64"
+  defaultValue="nil"
+>
+
+Metadata about numeric attributes of the subject. Example: `{"age": 30}`
+</ApiOptionRef>
+
+<ApiOptionRef 
+  name="SubjectCategoricalAttributes"
+  type="map[string]string"
+  defaultValue="nil"
+>
+
+Metadata about non-numeric attributes of the subject. Example: `{"loyalty_tier": "gold"}`
+</ApiOptionRef>
+
+<ApiOptionRef 
+  name="Action"
+  type="string"
+  defaultValue='""'
+>
+
+The action assigned by the bandit. Example: `"promo-20%-off"`
+</ApiOptionRef>
+
+<ApiOptionRef 
+  name="ActionNumericAttributes"
+  type="map[string]float64"
+  defaultValue="nil"
+>
+
+Metadata about numeric attributes of the assigned action. Example: `{"discount": 0.2}`
+</ApiOptionRef>
+
+<ApiOptionRef 
+  name="ActionCategoricalAttributes"
+  type="map[string]string"
+  defaultValue="nil"
+>
+
+Metadata about non-numeric attributes of the assigned action. Example: `{"promoTextColor": "white"}`
+</ApiOptionRef>
+
+<ApiOptionRef 
+  name="ActionProbability"
+  type="float64"
+  defaultValue="0"
+>
+
+The weight between 0 and 1 the bandit valued the assigned action. Example: `0.25`
+</ApiOptionRef>
+
+<ApiOptionRef 
+  name="ModelVersion"
+  type="string"
+  defaultValue='""'
+>
+
+Unique identifier for the version of the bandit parameters used. Example: `"v123"`
+</ApiOptionRef>
+
+## Querying the Bandit
+
+To query the bandit for an action, use the `GetBanditAction` method:
+
+```go
+attributes := eppoclient.Attributes{
+    NumericAttributes: map[string]float64{
+        "age": float64(user.Age),
+    },
+    CategoricalAttributes: map[string]string{
+        "country": user.Country,
+    },
+}
+
+actions := map[string]eppoclient.Attributes{
+    "nike": {
+        NumericAttributes: map[string]float64{
+            "brand_affinity": 2.3,
+        },
+        CategoricalAttributes: map[string]string{
+            "previously_purchased": "true",
+        },
+    },
+    "adidas": {
+        NumericAttributes: map[string]float64{
+            "brand_affinity": 0.2,
+        },
+        CategoricalAttributes: map[string]string{
+            "previously_purchased": "false",
+        },
+    },
+}
+
+result, err := client.GetBanditAction(
+    "shoe-bandit",
+    user.ID,
+    attributes,
+    actions,
+    "control",
+)
+if err != nil {
+    log.Printf("Error getting bandit action: %v", err)
+    return
+}
+
+if result.Action != "" {
+    showShoeAd(result.Action)
+} else {
+    showDefaultAd()
+}
+```
+
+### Subject Context
+
+The subject context contains contextual information about the subject that is independent of bandit actions.
+For example, the subject's age or country.
+
+The subject context has type `Attributes` which has two fields:
+- `NumericAttributes` (map[string]float64): A map of numeric attributes (such as "age")
+- `CategoricalAttributes` (map[string]string): A map of categorical attributes (such as "country")
```

Comment:
```suggestion
- `Numeric` (map[string]float64): A map of numeric attributes (such as "age")
- `Categorical` (map[string]string): A map of categorical attributes (such as "country")
```
Most examples that reference `NumericAttributes`/`CategoricalAttributes` are off